src/platform/mhv/secondary-nav/components/MhvSecondaryNavItem.jsx

```diff
@@ -31,6 +31,8 @@ const MhvSecondaryNavItem = ({
   const itemClass = classNames(
     'mhv-c-sec-nav-item',
     'vads-u-text-align--left',
+    'vads-u-display--flex',
```

Comment:
Note: We cannot remove these classes from the `mhv-c-sec-nav-bar-row` as they are needed on the row too. On newer browser the inheritance works correctly and these would not be needed.